generated/nidigital/nidigital/enums.py

```diff
@@ -6,69 +6,165 @@
 
 class BitOrder(Enum):
     MSB = 2500
+    r'''
+    The most significant bit is first. The first bit is in the 2n place, where n is the number of bits.
+    '''
     LSB = 2501
+    r'''
+    The least significant bit is first. The first bit is in the 20 place.
```

Comment:
Yeah, that doesn't make sense to me either.  Should I just drop everything after the first sentence for both of these descriptions?  Should this change also be made in the .NET help?

Reply:
It's supposed to be 2^n and 2^0. See https://zone.ni.com/reference/en-XX/help/375145e/ninetdigitalfx40ref/html/t_nationalinstruments_modularinstruments_nidigital_bitorder/

But yeah, you can drop that part. It's pretty descriptive without it. I filed an internal bug to fix the .NET help (1265897).

Reply:
OK, that makes sense.  Other options:
* Insert the carat character.
* Use restructured text directive.  This would cause it to be rendered correctly in the generated HTML, though the docstring would contain the funky RST, e.g. `` 2\ :superscript:`n` ``.

Reply:
I'm ok with nothing or the carat character. Doesn't seem worth bringing in the funky RST.

Reply:
Added a carat character.